### PR TITLE
Add post on docutils 0.18

### PR DESCRIPTION
**Pull request by agjohnson**
Also fix some issues with links on old posts.

I lost steam on the title and meta summary :chart_with_downwards_trend:

### CODE REVIEW

File: new-errors-in-recent-builds.rst [hunk 1]
```diff
@@ -0,0 +1,93 @@
+.. post:: November 2, 2021
+    :tags: builders
+    :author: Anthony
+    :location: GEG
+
+.. meta::
+    :description lang=en:
+        TBD
```
Comment (agjohnson) [inline new-errors-in-recent-builds.rst:8]: :construction:

File: new-errors-in-recent-builds.rst [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+
+New errors in recent builds
```
Comment (agjohnson) [inline new-errors-in-recent-builds.rst:10]: :construction:

**Comment by agjohnson** [comment]
@astrojuanlu figured I'd get this out for now as your time might be limited this week. You might have some feedback here as well. You've been spending time down here in the docutils mines as well. I lost steam on finishing this, feel free to run with this if you feel so inclined.

**Comment by astrojuanlu** [comment]
I think we reached peak irony: our blog cannot build because of docutils 0.18 😂 https://github.com/readthedocs/blog/runs/4087736364?check_suite_focus=true#step:5:10

File: new-errors-in-recent-builds.rst (hunk 2 above)
Comment (astrojuanlu) [reply new-errors-in-recent-builds.rst:10]: It's probably not perfect but I changed this to "Build errors with docutils 0.18". I think the body of the post does a great job at explaining what's happening in layman terms, but still clearly communicating the issue in the title in a non-time dependent way (i.e. avoid "new" and "recent") is necessary.

**Review by astrojuanlu — APPROVED** [review]
Thanks @agjohnson !
